modules/tensor_mechanics/src/materials/crystal_plasticity/CrystalPlasticityStressUpdateBase.C

```diff
@@ -203,8 +202,7 @@ CrystalPlasticityStressUpdateBase::transformHexagonalMillerBravisSlipSystems(
     const MooseUtils::DelimitedFileReader & reader)
 {
   const unsigned int miller_bravis_indices = 4;
-  std::vector<Real> miller_bravis_slip_direction, temporary_slip_direction, temporary_slip_plane;
-  miller_bravis_slip_direction.resize(miller_bravis_indices);
+  std::vector<Real> temporary_slip_direction, temporary_slip_plane;
```

Comment:
any reason those are not `RealVectorValue`? would be faster than std::vector and would support overloaded operators (for the matrix vector product below)

Reply:
I suppose this only applies to `temporary_slip_direction`